ide/zerobrane/packages/terrame.lua

```diff
@@ -62,7 +62,7 @@ local interpreter = {
             -- hack in Mac OS X
             if terrame == nil and ide.osname == "Macintosh" then
                 terrame = "/Applications/terrame.app/Contents/bin"
-            else
+            elseif terrame == nil then
                 DisplayOutputLn("Could not find TME_PATH. Is TerraME installed?")
 				return
```

Comment:
Please check and update the indentation of this file. Update it to use only tabs.